src/pages/home/assets/home.scss

```diff
@@ -1,57 +1,148 @@
-@import "~theme/base/variables";
+@import '~theme/base/variables';
+@import '~theme/base/utils';
 
-.homeContainer {
-  height: 100%;
-}
+$distance-height-variation-before: 20px;
+$distance-height-variation-after: 40px;
+
+// Sections with the text at the right
+@mixin isReverseSection() {
+    text-align: right;
+
+    header {
+        &:after {
+            right: 0;
+        }
+    }
 
-.mainHeaderBackground {
-  background: url('./main-header-background.png') no-repeat;
-  height: 100%;
-  width: 100%;
-  background-size: cover;
-  position: absolute;
-  z-index: -1; 
+    @media screen and (min-width: 768px) {
```

Comment:
Please use the `respond-to` mixing that we already have: https://github.com/Coding-Coach/coding-coach/blob/development/src/theme/base/_mediaqueries.scss#L36